.github/workflows/wheels-linux.yml

```diff
@@ -0,0 +1,37 @@
+name: wheels-linux
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
do we want to  build wheels on PRs too? Or just on master?

Reply:
we can revisit this later, but maybe add a gate for publishing the artifacts (always build the artifacts though)

Reply:
We'll probably want to build the wheels in regular testing, but we should probably have some logic so that we only store the artifacts and upload to PyPI when we're actually making a release. Not sure the best way to trigger that tho...

Maybe something like in this project: https://github.com/oxfordcontrol/qdldl-python/blob/master/.github/workflows/build.yml

```yaml
- name: Release to pypi
  if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
```